cmd/kubectl-package/command/tree_test.go

```diff
@@ -0,0 +1,57 @@
+package command_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"package-operator.run/package-operator/cmd/kubectl-package/command"
+)
+
+func TestTree(t *testing.T) {
+	t.Parallel()
+
+	t.Run("namespace scoped", func(t *testing.T) {
+		cmd := command.CobraRoot()
+		stdout := &bytes.Buffer{}
+		stderr := &bytes.Buffer{}
+		cmd.SetOut(stdout)
+		cmd.SetErr(stderr)
+		cmd.SetArgs([]string{"tree", "testdata"})
+
+		err := cmd.Execute()
+
+		require.Nil(t, err)
+		require.Len(t, stderr.String(), 0)
+
+		assert.Equal(t, `test-stub
+Package <namespace>/<name>
+└── Phase deploy
+    └── apps/v1, Kind=Deployment /test-stub-<name>
+`, stdout.String())
+	})
+
+	t.Run("cluster scoped", func(t *testing.T) {
+		cmd := command.CobraRoot()
+		stdout := &bytes.Buffer{}
+		stderr := &bytes.Buffer{}
+		cmd.SetOut(stdout)
+		cmd.SetErr(stderr)
+		cmd.SetArgs([]string{"tree", "testdata", "--cluster"})
+
+		err := cmd.Execute()
+
+		require.Nil(t, err)
+		require.Len(t, stderr.String(), 0)
+
+		assert.Equal(t, `test-stub
+ClusterPackage /<name>
```

Comment:
Maybe make a constant out of that? Threw me off when reading through it :D but no biggy

Reply:
fixed